docs/hdfs_connector.rst

```diff
@@ -95,13 +150,17 @@ You should see the following output::
 
 .. note:: If you want to run the Quickstart with Hive integration, before starting the connector,
```

Comment:
I agree. Added stop examples in S3's quickstart. I'll add the same here.

There wasn't a section here, and ideally I'd like to restrict the changes to the ones relevant to CLI. Still, after reading again I think the Hive part is pretty small. Not sure it justifies a new (sub)section. Not feeling strong about it either. I'll add it if you prefer it that way.